src/lightgbm/src/main/python/LightGBMRegressor.py

```diff
@@ -31,8 +32,33 @@ def saveNativeModel(self, sparkSession, filename):
         jsession = sql_ctx.sparkSession._jsparkSession
         self._java_obj.saveNativeModel(jsession, filename)
 
+    @staticmethod
+    def loadNativeModelFromFile(sparkSession, filename, labelColName="label",
+                                featuresColName="features", predictionColName="prediction"):
+        """
+        Load the model from a native LightGBM text file.
+        """
+        jsession = sparkSession._jsparkSession
+        ctx = sparkSession.sparkContext
+        loader = ctx._jvm.com.microsoft.ml.spark.LightGBMRegressionModel
+        java_model = loader.loadNativeModelFromFile(jsession, filename, labelColName,
+                                                    featuresColName, predictionColName)
+        return JavaParams._from_java(java_model)
+
+    @staticmethod
+    def loadNativeModelFromString(sparkSession, model, labelColName="label",
+                                  featuresColName="features", predictionColName="prediction"):
+        """
+        Load the model from a native LightGBM model string.
+        """
+        ctx = sparkSession.sparkContext
```

Comment:
likewise there is a python equivalent to SparkSession.getOrCreate. Its something like getActiveSession

Reply:
done!